CHANGELOG.md

```diff
@@ -13,10 +13,19 @@ does not match the expected response (200 for most requests and 202 for a few re
 trigger async operations). Previously the helpers in this package would return the response
 regardless of the status code, so you may need to update any code using helpers from this
 package to catch or to otherwise handle errors that you may encounter.
+- The Cumulus API Lambda function has now been configured with reserved concurrency to ensure
+availability in a high-concurrency environment. However, this also caps max concurrency which
+may result in throttling errors if trying to reach the Cumulus API multiple times in a short
+period. Reserved concurrency can be configured with the `archive_api_reserved_concurrency`
+terraform variable on the Cumulus module and increased if you are seeing throttling erorrs.
+The default reserved concurrency value is 8.
```

Comment:
It looks like it's set to 5 below?

Reply:
Oh I see. The default for `example` is 5 and 8 otherwise. Seems fine. Do any users deploy from example? Any reason we would want these to be the same?

Reply:
> Any reason we would want these to be the same?

I was thinking of lowering the example value more, as each account in our sandbox env will deduct from our account wide limit. 8 is a useful default for users, but we don't want to have 30 deployments all reserving 8 concurrency as that would reserve 24% of our account-wide limit.

> How should we test/verify that this is doing what we want?

What I did was deploy it and check on the AWS console that the lambda has the expected value under _Configuration > Concurrency_, which it did.

Reply:
Verified. Thanks for the info! I deployed to my stack and the `ApiEndpoints` shows a reserved concurrency of 2. Nice work!